packages/frontend/src/features/metricsCatalog/components/MetricTree/MetricTreeConnectedNode.tsx

```diff
@@ -84,6 +84,14 @@ const MetricTreeConnectedNode: React.FC<
         [totalQuery.data, data.timeFrame],
     ); // TODO: will it always be prev month?
 
+    const formattedValue = useMemo(() => {
+        if (totalQuery.data) {
+            const customFormat = getCustomFormat(totalQuery.data.metric);
+            return applyCustomFormat(totalQuery.data.value, customFormat);
```

Comment:
Could we have used formatItemValue ?
`formatItemValue(metric, totalQuery.data.value)`